mne/viz/_brain/surface.py

```diff
@@ -159,3 +160,30 @@ def load_curvature(self):
         color = 0.5 - (color - 0.5) / 3
         color = color[:, np.newaxis] * [1, 1, 1]
         self.grey_curv = color
+
+
+def mesh_edges(faces):
+    """Return a sparse matrix with edges as an adjacency matrix.
+
+    Parameters
+    ----------
+    faces : array of shape [n_triangles x 3]
+        The mesh faces
+
+    Returns
+    -------
+    edges : sparse matrix
+        The adjacency matrix
+    """
+    npoints = np.max(faces) + 1
+    nfaces = len(faces)
+    a, b, c = faces.T
+    edges = sparse.coo_matrix((np.ones(nfaces), (a, b)),
+                              shape=(npoints, npoints))
+    edges = edges + sparse.coo_matrix((np.ones(nfaces), (b, c)),
+                                      shape=(npoints, npoints))
+    edges = edges + sparse.coo_matrix((np.ones(nfaces), (c, a)),
+                                      shape=(npoints, npoints))
+    edges = edges + edges.T
+    edges = edges.tocoo()
+    return edges
```

Comment:
```
$ git grep "def mesh_edges"
mne/surface.py:def mesh_edges(tris):
```
Did you check to see if that one will work already? If so, then you shouldn't need this version, but rather import the other one with your `from ...surface import ...` line. There have been other duplicates like this in `_brain` before and I've been trying to remove them as I've found them. Let's try not to add more...